### PR TITLE
[Bugfix:InstructorUI] Uniform font on New Gradeable page

### LINKED ISSUE #10328: Make all text fields on the "Add Gradeable" page the same font
**Describe the bug**
On the instructor "Add Gradeable" page, most of the text input fields use the same font, while the gradeable title field uses a different one.

**Expected behavior**
The font should be uniform across all text fields on the page

**To Reproduce**
1. Login as instructor
2. Press "Add Gradeable" in sidebar
3. Type in the title field, then type in any other field

**Configuration**
 - OS: Windows
 - Browser Chrome

**Screenshots**
image

### PR DESCRIPTION
**Pull request by sadam-04**
### Please check if the PR fulfills these requirements:

* [X] Tests for the changes have been added/updated (if possible)
* [X] Documentation has been updated/added if relevant
* [X] Screenshots are attached to Github PR if visual/UI changes were made

Fixes #10328 

### What is the current behavior?
Some textboxes on the New Gradeable page are a monospace font, while others are not.
before

### What is the new behavior?
All text boxes are the same monospaced font. 
after

### Other information?
This is not a breaking change
I checked each textbox on the New Gradeable page from an instructor login

### CODE REVIEW

File: site/public/css/admin-gradeable.css [hunk 1]
```diff
@@ -225,6 +225,13 @@ div.btn-container {
     border-radius: 5px;
 }
 
+/* Make all text input fields on AdminGradeableCreate monospace. TODO: decide on site-wide font policy
+#g_title, #instructions_url, #g_id, #discussion_thread_id, #team_size_max, #vcs_path, #external_repo, #vcs_subdirectory_input {
+    font-family: monospace;
+}
+*/
+
+
```
Comment (ArnavPawar) [inline site/public/css/admin-gradeable.css:234]: You dont need these comments

**Review by ziesski — APPROVED** [review]
Reasonable change, would have been nicer if you included screenshot of expected output as well, so that testers could verify if it matches the expected output.

**Comment by williamjallen** [comment]
Can you please add before/after screenshots directly to the PR description?  The PR description is the long-term documentation of this change, and we want people to be able to understand what this change does without running the code locally.

**Review by williamjallen — APPROVED** [review]

**Comment by bmcutler** [comment]
@sadam-04 
Please add the screenshots requested by @williamjallen 
Thanks

**Pull request merged**